### ISSUE TITLE
Fatal error when building 

### ISSUE THREAD

**Issue by RaedRokh** [openssl/openssl#20627]
Trying to build openssl following a youtube video https://www.youtube.com/watch?v=PMHEoBkxYaQ&t=179s

I get this error after using nmake test or nmake install 

`Could Not Find B:\raed\Documents\openssl-3.1.0-alpha1\libcrypto-3-x64.*
NMAKE : fatal error U1077: 'cmd /C ""link" /nologo /debug /dll  /nologo /debug @C:\Users\raed\AppData\Local\Temp\nm4F49.tmp /implib:libcrypto.lib || (DEL /Q libcrypto-3-x64.* libcrypto.lib & EXIT 1)"' : return code '0x1'
Stop.
NMAKE : fatal error U1077: '"B:\program files\Microsoft Visual Studio\2022\Professional\VC\Tools\MSVC\14.35.32215\bin\HostX86\x86\nmake.exe" /                   _build_libs' : return code '0x2'
Stop.
`

Is there any way to fix this issue? and if there's any beginner-friendly tutorial to install openssl and use it in Visual studio for a c project.

Thank you & have a nice day.

**Comment by mattcaswell** [comment]
Please can you provide further details of the commands you entered prior to this failure.

There is a quick start guide for building OpenSSL on Windows in the file NOTES-WINDOWS.md in the downloaded files:

https://github.com/openssl/openssl/blob/openssl-3.1/NOTES-WINDOWS.md

It looks like you are using the 3.1.0-alpha1 version. I'd recommend that you actually use the final 3.1.0 version downloadable from here:

https://www.openssl.org/source/

**Comment by hello-willy** [comment]
I also encountered this problem，build log as below:
```
IF EXIST .manifest DEL /F /Q .manifest
        IF EXIST libcrypto-3-x64.dll DEL /F /Q libcrypto-3-x64.dll
        cmd /C ""link" /nologo /debug /dll  /nologo /debug @C:\Users\joba_\AppData\Local\Temp\nm6A39.tmp /implib:libcrypto.lib || (DEL /Q libcrypto-3-x64.* libcrypto.lib & EXIT 1)"
link: extra operand '/dll'
Try 'link --help' for more information.
Could Not Find D:\SourceCode\openssl-3.1.1\libcrypto-3-x64.*
NMAKE : fatal error U1077: “cmd”: return code “0x1”
Stop.
NMAKE : fatal error U1077: “"C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Tools\MSVC\14.29.30133\bin\HostX64\x64\nmake.exe"”: returen code “0x2”
Stop.
```
Thank you!!!

**Comment by nhorman** [comment]
marking as inactive, will be closed at the completion of 3.4 dev without further input
